server/src/server.js

```diff
@@ -73,6 +73,11 @@ app.use(function (req, res, next) {
   next();
 });
 
+app.use(function (req, res, next) {
+  req.allowExport = config.allowExport;
```

Comment:
It should be possible to use `req.config.allowExport`, and not copy the variable over